app/models/procedures_filter.rb

```diff
@@ -35,6 +35,10 @@ def tags
     params[:tags].compact_blank.uniq if params[:tags].present?
   end
 
+  def is_template
+    params[:is_template].in?(["0", '1', false, true]) ? ActiveRecord::Type::Boolean.new.cast(params[:is_template]) : nil
+  end
```

Comment:
En ruby la convention fait qu'on utilise des méthodes se terminant par `?` pour renvoyer un booléen, plutôt qu'un `is_` ou `has_`
aussi le cast fait justement ce que tu fais avec le `.in?` donc je propose cette méthode à la place : 

```suggestion
  def template?
    ActiveRecord::Type::Boolean.new.cast(params[:is_template])
  end
```